contracts/libraries/SafeERC677.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.8.9;
+pragma abicoder v1;
+import "../token/IERC677.sol";
+
+library SafeERC677 {
+  function safeTransferAndCall(
+    IERC677 token,
+    address to,
+    uint256 value,
+    bytes memory data
+  ) internal returns (bool) {
```

Comment:
I think we should not return anything here to be consistent with `safeTransfer` - it can only ever be true and wastes some small amount of gas to return a value for no reason,.

Reply:
Yeah makes sense. However I had to add the return [here](https://github.com/cardstack/card-pay-protocol/pull/178/commits/ba85994b3f8167c1173d0ab88c3611edbf36ce52#diff-5e88e4612774f782757ca331921061bef2315fbe635859ef415526c150ddc6b8R111) otherwise a couple of PrepaidCard tests break with ` Error: VM Exception while processing transaction: reverted with reason string 'safe transaction was reverted'`.

Reply:
Perfect, we actually have another audit recommendation in general that we stop using pointless return statements like that but fine for this PR!